packages/dd-trace/src/config.js

```diff
@@ -508,6 +508,17 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       true
     )
 
+    const DD_EXPERIMENTAL_API_SECURITY_ENABLED = coalesce(
+      process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED && isTrue(process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED),
```

Comment:
Don't we need `options.appsec?.apiSecurity?.enabled` ?
Imo, or we need the options, or we can remove the option for requestSampling.